package.json

```diff
@@ -96,6 +96,7 @@
   "prettier": {
     "jsdocCommentLineStrategy": "keep",
     "jsdocPreferCodeFences": true,
+    "jsdocTagsOrder": "{\"template\": 24.5}",
```

Comment:
I don't understand this and therefore fear it. I'm a simple man. Please help me understand why we want this here.

Reply:
heh, well, I would have explained myself, but it's `package.json`, so.

https://github.com/hosseinmd/prettier-plugin-jsdoc/issues/226